eox_audit_model/models.py

```diff
@@ -163,25 +164,18 @@ def execute_action(cls, action, method, parameters, notes=None):
             status = Status.FAIL
             raise error
         finally:
-            if getattr(settings, 'ALLOW_EOX_AUDIT_MODEL', False):
-                audit_register = cls.objects.create(
+            if getattr(settings, 'ALLOW_EOX_AUDIT_MODEL', True):
```

Comment:
why true is the new default?

Reply:
yes, it is, this is a different ticket https://edunext.atlassian.net/browse/PS2021-226